app/build.gradle

```diff
@@ -20,23 +20,38 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    flavorDimensions "default"
+    productFlavors {
```

Comment:
just to understand, why did you create a mock flavor? As far as I understood, is this change changing the applicationId, right?

Reply:
I intended to create mocks for testing, e.g. an injected mock repository with fake data, and that would be in the mock flavor, but unfortunately I didn't get time to implement tests, so I left it here just for future work.